arches/app/models/graph.py

```diff
@@ -1586,15 +1589,18 @@ def publish(self, notes=None):
         Adds a row to the GraphPublication table
         Assigns GraphPublication id to Graph
         """
-        publication = models.GraphPublication.objects.create(
-            graph=self,
-            serialized_graph=JSONSerializer().serialize(self, force_recalculation=True),
-            notes=notes,
-        )
-        publication.save()
+        try:
```

Comment:
when will this fail? is it possible / worth it to replace the try/except with a check of some kind?